app/models/orphan.rb

```diff
@@ -15,7 +15,7 @@ class Orphan < ActiveRecord::Base
 
   validates :name, presence: true,
             uniqueness: { scope: [:family_name, :mother_name, :father_given_name],
-                          message: 'An orphan with this name, father, mother & family name is already in the database.' }
+                          message: 'taken: an orphan with this name, father, mother & family name is already in the database.' }
```

Comment:
The error message is automatically prepended with the attribute name - "Name" in this case. It is possible to override this behaviour, but only in a way that would affect all error messages. Changing this message from "Name An orphan with this name..." to "Name taken: an orphan with this name..." seems like an acceptable compromise.